### PR TITLE
feat(rate-tables): Rate Table 1

### PR DESCRIPTION
**Pull request by PrathamPatel232**
image

### CODE REVIEW

**Comment by PrathamPatel232** [comment]
image

File: libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.html [hunk 1]
```diff
@@ -1 +1,19 @@
-<p>rate-tables works!</p>
+<section class="main-div">
+  <section class="justify_content border parent flex">
+    <h3 class="parent_left white">GOLD PRODUCTS</h3>
+    <p class="parent_right white">SELL</p>
```
Comment (keyurboss) [inline libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.html:4]:
```suggestion
    <h3 class="parent_left white">{{expect input from parent}}</h3>
    <p class="parent_right white">{{expect input from parent}}</p>
```

File: libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  border: 1px solid black;
+}
+
+.main-div {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: Open Sans, Arial, Helvetica, sans-serif;
```
Comment (keyurboss) [inline libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.scss:29]:
```suggestion
  font-family: var(--font-family);
```

File: libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  --header-back-color: #bc9039;
+  --header-font-color: #ffffff;
+  --header-border-color: #bc9039;
+  --data-back-color: #ffffff;
+  --data-font-color: #ffffff;
+  --rate-high-back-color: #00ff40;
+  --rate-low-back-color: #ff0000;
+  --rate-high-font-color: #ffffff;
+  --rate-low-font-color: #ffffff;
+  --data-border-color: #000;
```
Comment (keyurboss) [inline libs/rate-tables/src/lib/rate-tables-1/rate-tables-1.component.scss:11]:
```suggestion
  --data-border-color: #000;
    --font-family: Open Sans, Arial, Helvetica, sans-serif;
```

**Review by keyurboss — COMMENTED** [review]
Remove Fixed Width 
Make it Resizable Also
Header and data section should have same left right margin

**Pull request merged**